CHANGES.md

```diff
@@ -1,15 +1,14 @@
 ## Unreleased
-- Remove unused values to help ci(@bikallem #873)
 - New curl based clients (@rgrinberg #813)
   + cohttp-curl-lwt for an Lwt backend
   + cohttp-curl-async for an Async backend
 - Completely new Parsing layers for servers (@anuragsoni #819)
   + Cohttp now uses an optimized parser for requests.
-  + The new parser produces much less temporary buffers during read operations in servers.
+  + The new parser produces much less temporary buffers during read operations
+    in servers.
 - Faster header comparison (gasche #818)
-- Introduce http package containing common signatures and structures useful for compatibility with cohttp - and no dependencies (rgrinberg #812)
-- Faster Request/Response comparison (rgrinberg #814)
-- Use raise_notrace in header short circuiting exceptions (rgrinberg #802)
```

Comment:
Why not leaving the optimizations?

Reply:
You mean the one about `raise_notrace`? Honestly it's so minor it's hardly worth mentioning. If you'd like, I can bring it back though.

Reply:
I meant both,
```
- Faster Request/Response comparison (rgrinberg #814)
- Use raise_notrace in header short circuiting exceptions (rgrinberg #802)
```
but it could also go, more compactly below